_source/_plugins/youtube_tag.rb

```diff
@@ -0,0 +1,20 @@
+module Jekyll
+  # A simple tag for YouTube include using an iframe
+    class YoutubeTag < Liquid::Tag
+
+      def initialize(name, id, tokens)
+        super
+        @id = id
+      end
+
+      def render(context)
+        %(<p>
```

Comment:
I think this should be changed to a div. Here's the raw HTML I used in a post this summer:

```
<div style="text-align: center; margin-bottom: 1.25rem">
<iframe width="700" height="394" style="max-width: 100%" src="https://www.youtube.com/embed/SQFl7ggNYIE" title="Kubernetes to the Cloud with Spring Boot and JHipster" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
</div>
```